File: 01 Introdução/2160 Nome no Formulario/2160.java
import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    String name = sc.nextLine();
    if (80 >= name.length()) {
      System.out.println("YES");
    } else {
      System.out.println("NO");
    }
    sc.close();
  }
}

// https://www.beecrowd.com.br/judge/pt/problems/view/2160
// Ranking 00637º, Time 0.213, Submiss Date 01/04/2023 20:27:50 (Java 14)

File: 01 Introdução/2160 Nome no Formulario/2160.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
  string name;
  getline(cin, name);

  if (name.length() <= 80)
  {
    cout << "YES" << endl;
  }
  else
  {
    cout << "NO" << endl;
  }
  return 0;
}

// https://www.beecrowd.com.br/judge/pt/problems/view/2160
// Ranking 02223º, Time 0.000, Submiss Date 01/04/2023 21:11:55 (C++17)